sql/comments_database_setup.sql

```diff
@@ -289,3 +289,15 @@ CREATE TABLE semesterInfo (
   updatedAt TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
   PRIMARY KEY (universityId, instanceId)
 );
+
+CREATE TABLE courseMetaData (
+    courseId VARCHAR(50) NOT NULL,     
+    instanceId VARCHAR(50) NOT NULL,     
+    lectureDay VARCHAR(20) NOT NULL,     
+    venue VARCHAR(10),  
```

Comment:
There can be multiple lecture entries. Eg Mon 3-4PM, Tue 10-11AM.
How will that be handled?